src/WalkingModule/app/robots/iCubGenova04/dcm_walking/step_adaptation/stepAdaptation.ini

```diff
@@ -0,0 +1,36 @@
+#The gain of cost function components(alpha1,...)
+next_zmp_position_weight     (1.0, 1.0)
+next_dcm_offset_weight       (100.0, 100.0)
+sigma_weight                  0.05
+
+step_duration_tolerance       0.0
+
+# front (x positive), back (x negative), left (y positive) and right (y negative)
+next_zmp_constraint_bound_left_foot           (0.01, 0.00001, 0.08, 0.00001)
+next_zmp_constraint_bound_right_foot          (0.01, 0.00001, 0.0001,  0.08)
+
+#delta=p2-p1
```

Comment:
What are p2 and p1?

Reply:
They were related to heel-toe motion that now we don't use them inside the code! Removed!